### ISSUE TITLE
Are you using this with a full local installation?

### ISSUE THREAD

**Issue by cagnulein** [technyon/nuki_hub#274]
Hi, thanks for this great application!
I just installed and it works flawlessly!

Just a thought: i'm not a fan of closed ecosystem so i would like to DON'T use the nuki app at all for my phone and my wife's phone and use this and the hass integration to do everything.

Does anyone use this in a similar scenario? My idea is that i don't want any IoT device dependent from the cloud or from any 3rd party cloud service 

I'm planning to donate to this project in the next weeks if i will keep the nuki device for my house

Thanks!

**Comment by bartsnijder** [comment]
The Nuki app is standby for any possible firmware updates, but other than that the NUKI is indeed used through nuki_hub only (through mqtt/ domotica software). Works flawlessly.

**Comment by cagnulein** [comment]
ok so how could we handle the bluetooth authtentication in this scenario without the Nuki app? using the HASS state for each person?

**Comment by bartsnijder** [comment]
I do not follow what you mean. The bridge is connected/authenticated to the lock through bluetooth (as only device). The bridge has now become your primary interface (through MQTT). How you deal with e.g. multi-users, the ability to see the locked status and/or (un)lock at different times has now shifted to your domotica software.  I do not us HASS (but Openhab), but elaborating on this might be better done on a HASS forum I guess.

**Comment by cagnulein** [comment]
Yes indeed "How you deal with e.g. multi-users, the ability to see the locked status and/or (un)lock at different times has now shifted to your domotica software."

And so you asked to my question, you're using the ability of the domotica software to unlock or lock the nuki based on certain event. Got it.

so @bartsnijder are you using the nuki without the nuki app at all on your setup?

**Comment by bartsnijder** [comment]
In the daily setup: no I do not use the NUKI app.
But I do have the app standby for possible firmware updates.

**Comment by cagnulein** [comment]
ok thanks for the clarification!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 23 gen 2024 alle 12:08 bartsnijder ***@***.***>
ha scritto:

**Comment by bartsnijder** [comment]
I use a few means to automatically open the lock
* BLE presence (of NUKI hub): I do limit based on BLE signal strength
* Mobile phone and/or car connected to my LAN on a specific AP (+signal strength) (this does have some extra delay)

**Comment by cagnulein** [comment]
ok so you have also a nuki hub...

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 23 gen 2024 alle 13:30 bartsnijder ***@***.***>
ha scritto:

**Comment by bartsnijder** [comment]
No, I edited the command. By accident I named it a HUB. I only have the
bridge.

Op di 23 jan 2024 om 13:32 schreef Roberto Viola ***@***.***>:

**Comment by technyon** [comment]
The app on the smartphone is not necessarily cloud connected. If you use it without the NUKI Bridge, it directly connects to the lock via bluetooth without communicating with the cloud.

**Comment by cagnulein** [comment]
> No, I edited the reply. By accident I named it a HUB. I only have the bridge.

sorry, just to understand correctly, you mean you only have the ESP32 with the nuki hub? if so, does it expose any data from the bluetooth devices around it?



> The app on the smartphone is not necessarily cloud connected. If you use it without the NUKI Bridge, it directly connects to the lock via bluetooth without communicating with the cloud.

hah i didn't know this! thanks

**Comment by bartsnijder** [comment]
> > No, I edited the reply. By accident I named it a HUB. I only have the bridge.
> 
> sorry, just to understand correctly, you mean you only have the ESP32 with the nuki hub? if so, does it expose any data from the bluetooth devices around it?

Yes it does, do you have an MQTT browser/viewer (like mqtt explorer). It states all the BLE device found (including signal strength). I think your phone BLE will not be visible by default though; might need an app for that. I use different BLE devices as keys.

**Comment by cagnulein** [comment]
@bartsnijder that's great! i didn't know this! I'm installing https://community.home-assistant.io/t/addon-mqtt-explorer-new-version/603739 (for future reference) to check it and I will report it back. If it's so I guess I'm ready to give it a go! Thanks!

**Comment by technyon** [comment]
> Yes it does, do you have an MQTT browser/viewer (like mqtt explorer). It states all the BLE device found (including signal strength). I think your phone BLE will not be visible by default though; might need an app for that. I use different BLE devices as keys.

Correct, you have to install an app that broadcasts ble beacons for your phone to become visible for NUKI Hub. Also, it has to broadcast a name, otherwise the beacon will be ignored.

**Comment by cagnulein** [comment]
> Correct, you have to install an app that broadcasts ble beacons for your phone to become visible for NUKI Hub. Also, it has to broadcast a name, otherwise the beacon will be ignored.

that's awesome @technyon thanks! do you know if it also possible on ios? which app do you suggest?

**Comment by technyon** [comment]
I'm not using an app myself, but a Siemens G-Tag. I think many people use the Home Assistant companion app, but that only makes sense if you're using home assistant. Just search on the play store ("beacon simulator"), there are a few apps that can do it.

I'm not sure about IOS, I'm strictly an android user :).

**Comment by iranl** [comment]
In accordance with https://docs.github.com/en/get-started/using-github/communicating-on-github#scenarios-for-github-discussions this open-ended conversation is more suitable for the discussions section imho.

@technyon: Now that GitHub discussions is enabled on the repository I propose we convert this issue to a discussion.
